src/tools/kdb/cmdline.cpp

```diff
@@ -78,62 +78,62 @@ Cmdline::Cmdline (int argc,
 	{
 		option o = {"all", no_argument, 0, 'a'};
 		long_options.push_back(o);
-		helpText += "-a --all                 consider all keys\n";
+		helpText += "-a --all                 Consider all of the keys.\n";
 	}
 	if (acceptedOptions.find('d')!=string::npos)
 	{
 		option o = {"debug", no_argument, 0, 'd'};
 		long_options.push_back(o);
-		helpText += "-d --debug               give debug information or ask debug questions (in interactive mode)\n";
+		helpText += "-d --debug               Give debug information or ask debug questions (in interactive mode).\n";
 	}
 	if (acceptedOptions.find('f')!=string::npos)
 	{
 		option o = {"force", no_argument, 0, 'f'};
 		long_options.push_back(o);
-		helpText += "-f --force               force the action to be done\n";
+		helpText += "-f --force               Force the action to be done.\n";
 	}
 	if (acceptedOptions.find('l')!=string::npos)
 	{
 		option o = {"load", no_argument, 0, 'f'};
 		long_options.push_back(o);
-		helpText += "-l --load                load plugin even if system/elektra is available\n";
+		helpText += "-l --load                Load plugin even if system/elektra is available\n";
 	}
 	if (acceptedOptions.find('h')!=string::npos)
 	{
 		option o = {"human-readable", no_argument, 0, 'h'};
 		long_options.push_back(o);
-		helpText += "-h --human-readable      print numbers in an human readable way\n";
+		helpText += "-h --human-readable      Print numbers in an human readable way\n";
 	}
 	if (acceptedOptions.find('H')!=string::npos)
 	{
 		option o = {"help", no_argument, 0, 'H'};
 		long_options.push_back(o);
-		helpText += "-H --help                print help text\n";
+		helpText += "-H --help                Print help text.\n";
 	}
 	if (acceptedOptions.find('i')!=string::npos)
 	{
 		option o = {"interactive", no_argument, 0, 'i'};
 		long_options.push_back(o);
-		helpText += "-i --interactive         instead of passing all information by parameters\n";
-		helpText += "                         ask the user interactively\n";
+		helpText += "-i --interactive         Instead of passing all information by parameters\n";
+		helpText += "                         ask the user interactively.\n";
 	}
 	if (acceptedOptions.find('n')!=string::npos)
 	{
 		option o = {"no-newline", no_argument, 0, 'n'};
 		long_options.push_back(o);
-		helpText += "-n --no-newline          suppress the newline at the end of the output\n";
+		helpText += "-n --no-newline          Suppress the newline at the end of the output.\n";
 	}
 	if (acceptedOptions.find('t')!=string::npos)
 	{
 		option o = {"test", no_argument, 0, 't'};
 		long_options.push_back(o);
-		helpText += "-t --test                test\n";
+		helpText += "-t --test                Test.\n";
```

Comment:
Nice fixes!

Maybe you can find out more about these options. "Test." is not really descriptive.